docs/modules/ROOT/pages/getting-started/your-first-query.adoc

```diff
@@ -0,0 +1,80 @@
+= Your first query
+
+In this guide, you'll create a simple Cypher query to match and return Movies using the Cypher Builder:
+
+```cypher
+MATCH(m:Movie)
+RETURN m
+```
+
+== Matching nodes
+First, you need to define a node variable in your script. You can do it by creating a new instance of `Cypher.Node`:
+
+```javascript
+const movieNode = new Cypher.Node({
+  labels: ["Movie"]
+});
+```
+
+Then, you need to create a `MATCH` clause and pass the node variable as an argument:
+
+```javascript
+const clause = new Cypher.Match(movieNode);
+```
+
+== Returning the variable
+
+The second part of the query is the `RETURN` of the Node variables, you can do this with the `.return` method in the match clause:
```

Comment:
In terms of style guide, we should consider whether we start writing `MATCH`/`RETURN`/etc clause or without formatting

```suggestion
. To `RETURN` the Node variables, use the `.return` method in the `MATCH` clause:
```

Reply:
mmm, the problem is that `MATCH` refers to the result cypher, while lowercase match clause refers to the javascript code (Bit confusing :sweat: )